src/EditorFeatures/CSharpTest/QuickInfo/SyntacticQuickInfoSourceTests.cs

```diff
@@ -281,21 +280,25 @@ private QuickInfoProvider CreateProvider(TestWorkspace workspace)
         protected override async Task AssertContentIsAsync(
             TestWorkspace workspace,
             Document document,
+            ITextSnapshot snapshot,
             int position,
             string expectedContent,
             string expectedDocumentationComment = null)
         {
             var provider = CreateProvider(workspace);
             var info = await provider.GetQuickInfoAsync(new QuickInfoContext(document, position, CancellationToken.None));
             Assert.NotNull(info);
-
             Assert.NotEqual(0, info.RelatedSpans.Length);
-            var tabSize = document.Project.Solution.Workspace.Options.GetOption(Microsoft.CodeAnalysis.Formatting.FormattingOptions.TabSize, document.Project.Language);
-            var text = await document.GetTextAsync();
 
-            var classifiedSpans = info.RelatedSpans.Select(s => new ClassifiedSpan(ClassificationTypeNames.Text, s));
-            var spans = IndentationHelper.GetSpansWithAlignedIndentation(text, classifiedSpans.ToImmutableArray(), tabSize);
-            var actualText = string.Concat(spans.Select(s => text.GetSubText(s.TextSpan).ToString()));
+            var trackingSpan = new Mock<ITrackingSpan>();
```

Comment:
You may want to say MockBehavior.Strict; otherwise anybody who calls into this Mock might get something strange, which isn't good. Since you have a buffer creating a real one is also really easy too.